-- let_err4 --
let x = 1
let x = 1
y: x
-- exp --
error: x redeclared in same scope
